lerna.json

```diff
@@ -5,6 +5,6 @@
   "devDependencies": {
     "lerna": "^3.4.3"
   },
-  "version": "0.0.4",
+  "version": "0.0.4-alpha",
```

Comment:
Why this version change in this PR?

Reply:
I published a testing version just to make sure it works. Once we merged the changes I'll publish `0.0.5`. I committed the version change here to make it explicit. Shouldn't matter much?

Reply:
Yeah I don't think it matters much, just think we should generally try to avoid that in PRs to avoid merge conflicts.

Reply:
fair point, will update